src/main/flight/autopilot.h

```diff
@@ -19,12 +19,16 @@
 
 #include "pg/autopilot.h"
 #include "flight/pid.h"
+#include "io/gps.h"
+
+extern float posHoldAngle[ANGLE_INDEX_COUNT]; // NOTE: ANGLES ARE IN CENTIDEGREES
```

Comment:
reading the name, it looks like this is part of posHold, not autopilot.

Reply:
Freudian slip ;-)
changed to autopilotAngle